edelphi/src/main/java/fi/metatavu/edelphi/pages/panel/admin/report/util/ChartModelProvider.java

```diff
@@ -638,7 +662,7 @@ public static Chart createBubbleChart(String chartCaption, String xLabel, List<S
 
     // Y-Axis
 
-    Map<Double, String> yTickLabelMap = new HashMap<Double, String>();
+    Map<Double, String> yTickLabelMap = new HashMap<>();
     for (int i = 0, l = yTickLabels.size(); i < l; i++) {
       yTickLabelMap.put(new Double(i), yTickLabels.get(i));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this "Double" constructor [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2129)